### DISCUSSION TITLE
Can someone please send me some ETH on the Rinkeby chain?

### DISCUSSION THREAD

**Discussion by ChristianG77** [smartcontractkit/full-blockchain-solidity-course-py#221] [Q&A] [answered]
Hi everyone,

Could some please send me some ETH on the Rinkeby chain?

I've tried to post on Twitter but nothing comes through.  

I've used the Kovan chain as a substitute for most of the course, but now in Lesson 11, OpenSea requires ETH on Rinkeby.

Any help would be much appreciated.

My address is:  0x2e3C5DaC9dEad2eE59e375Ad30a48b51D4587C1E

Thanks.

**Comment by PatrickAlphaC** [comment, marked as answer]
Can you tell me if the faucets listed here dont work? https://docs.chain.link/docs/link-token-contracts/#rinkeby

**Reply by ChristianG77** [reply]
Hi @PatrickAlphaC thanks for getting back to me.

I tried both ETH links in the link you provided above and both did not work for me unfortunately.

**Reply by PatrickAlphaC** [reply]
https://rinkeby.etherscan.io/tx/0xf30c2ac1f836a4fdd44b070a41525b2fc2e365d14a2f034480c0bea055ae34d2

**Reply by ChristianG77** [reply]
Thanks @PatrickAlphaC  much appreciated!
